app/components/scoreboard_summary_component.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class ScoreboardSummaryComponent < ViewComponent::Base
+  attr_reader :podium
+
+  include ApplicationHelper
+
+  def initialize(podium:)
+    @podium = podium
+  end
+
+  def other_schools?
+    podium.positions.count > 1
+  end
+
+  def school
+    podium.school
+  end
+
+  def scoreboard
+    podium.scoreboard
+  end
+
+  def limit
+    5
+  end
+
+  def observations
+    scope = other_schools? ? scoreboard.observations : Observation
```

Comment:
Defaulting to any activity across EnergySparks is fine, but just noting we might want to review whether we want to fall back to what other schools in their group are doing as a more directly relevant set of activities. That can come in the new year though after we've got some feedback.

Reply:
OK, sounds like a plan